index.html

```diff
@@ -51,22 +56,203 @@ <h1 id="principal-title" class="color-blue fonts">I’m Luis Emilio. <br> Glad t
+                            </div>
+                        </div>
+    </section>
+    <div class="headline">
+        <h2 id="principal-title" class="color-blue fonts">About Myself
+        </h2>
+        <p class="p-headline fonts">I’m a software developer! I can help you build a product , feature or website Look
+            through some of my work and experience! If you like what you see and have a project you need coded, don’t
+            hesitate to contact me.</p>
+        <p class="linkheadline fonts">LET’S CONNECT</p>
+
+
+        <ul class="social-icons-set">
+            <li>
+                <a href="https://twitter.com/lerfast"><img src="./vectors/twitter Vector.svg" alt="twitter-icon"></a>
+            </li>
+            <li>
+                <a href="https://www.linkedin.com/in/luis-emilio-rojas-rojas-6ba90a8a/"><img
+                        src="./vectors/Linkedin icon (1).png" alt="linkedin-icon"></a>
+            </li>
+            <li>
+                <a href="https://medium.com/"><img src="./vectors/middle.png" alt="medium-icon"></a>
+            </li>
+            <li>
+                <a href="https://github.com/lerfast"><img src="./vectors/github logo.png" alt="github-icon"></a>
+            </li>
+            <li>
+                <a href="https://www.angellist.com/"><img src="./vectors/angellist-icon.png" alt="angellist-icon"></a>
+            </li>
+
+        </ul>
+
+    </div>
+    <button type="button" class="see-btn2">Get my resume</button>
+    <div class="header2">
+
+        <p class="languages">Languages</p>
+
+        <nav>
+            <button type="button" class="button2">
+                <img src="./vectors/tertiary.png" alt="nav button">
+            </button>
+        </nav>
+    </div>
+
+    <div class="languages-container">
+        <div>
+            <div class="containerJS-style">
+                <img src="./vectors/Ellipse 1.png" alt="JAVASCRIPT logo">
+                <p class="languages-container-font">JavaScript</p>
+            </div>
+            <div class="containerHTML-style">
+                <img src="./vectors/HTML Ellipse 1.png" alt="HTML logo">
+                <p class="languages-container-font">HTML</p>
+            </div>
+            <div class="containerCSS-style">
+                <img src="./vectors/css Ellipse 1.png" alt="CSS logo">
+                <p class="languages-container-font">CSS</p>
+            </div>
+
+        </div>
+
+
+    </div>
+
+    <div class="header2">
+
+
+        <p class="languages">Frameworks</p>
+
+        <nav>
+            <button type="button" class="button2">
+                <img src="./vectors/tertiary.png" alt="nav button">
+            </button>
+        </nav>
+
+
+    </div>
+
+
+    <div class="header2">
+        <p class="languages">Skills</p>
+
+        <nav>
+            <button type="button" class="button2">
+                <img src="./vectors/tertiary.png" alt="nav button">
+            </button>
+        </nav>
+    </div>
```

Comment:
- [ ] These categories ("Languages", "Frameworks" etc) should be listed as list items `<li>`s inside a `<ul>` tag and the "Languages" (e.g., "JavasScript", "HTML" and "CSS") should be laid out as `<li>`s inside a `<ul>` as well, currently you are using `<div>s` which is not semantically proper and doesn't follow best practices, please fix that.